REVIEWING.md

```diff
@@ -0,0 +1,28 @@
+# Reviewing Yari changes
+
+This document provides information on how to review changes to the yari repo.
+
+## Before you start
+
+Set up the yari repo and the corresponding [content repo](https://github.com/mdn/content) locally, as described in the [Yari quickstart](README.md#quickstart) guide. Once you've got them successfully set up, run the `yarn` and `yarn dev` commands to update your fork with the latest packages and start the MDN test site running locally on `localhost:3000`.
+
+`yarn dev` is slower to execute than `yarn start` but it makes sure `client/build/` is clean, and it also reloads the SSR part of the site (it does a fresh Webpack build). This might matter after you've run `git pull` & `yarn`, since certain packages might be upgraded and it's always good to test with a clean, up-to-date build.
```

Comment:
```suggestion
`yarn dev` is slower to execute than `yarn start` but it makes sure `client/build/` is clean, and it also reloads the SSR part of the site (it does a fresh `webpack` build). This might matter after you've run `git pull` & `yarn`, since certain packages might be upgraded and it's always good to test with a clean, up-to-date build.
```

Reply:
fixed